src/lib/db/DatabaseManager.ts

```diff
@@ -1,22 +1,21 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
delete if unused

Reply:
It's used: eslint configuration code has to be wrapped in comments by default